ci-operator/config/openshift/hive/openshift-hive-master__periodic.yaml

```diff
@@ -39,7 +39,7 @@ tests:
     owner: openshift-ci
     product: ocp
     timeout: 1h0m0s
-    version: "4.19"
+    version: "4.20"
```

Comment:
This just informs which DPTP clusterpool we pull the hub from. It's a correct change, but not related to the k8s bump effort. (Technically this should probably have been done via HIVE-2848 / #66708, but I'm guessing the DPTP clusterpool wasn't ready yet.) Would prefer to see it in a separate PR to keep the lines clean.